cache_path_test.go

```diff
@@ -290,12 +345,32 @@ func Test_normalizeIndicatorByPath(t *testing.T) {
 		return
 	}
 
+	defer func() {
+		if err := os.RemoveAll(tmpDir); err != nil {
+			log.Warnf("failed to remove directory, error: %s", err)
+		}
+	}()
+
 	pths := map[string]string{
-		filepath.Join(tmpDir, "subdir", "file1"): "",
-		filepath.Join(tmpDir, "subdir", "file2"): "",
+		filepath.Join(tmpDir, "subdir", "file1"):          "",
+		filepath.Join(tmpDir, "subdir", "file2"):          "",
+		filepath.Join(tmpDir, "dir_with_symlink", "file"): "",
 	}
 	createDirStruct(t, pths)
 
+	linkFilePath := filepath.Join(tmpDir, "dir_with_symlink", "link_file")
```

Comment:
pls, define them before the os.Symlink calls to make it easier see which one used where just like you did here: https://github.com/bitrise-steplib/steps-cache-push/pull/33/files#diff-2d843efe8633410c6009ca4cb528e2bdR264